engine/src/main/java/org/terasology/logic/console/commands/CoreCommands.java

```diff
@@ -139,14 +138,56 @@
     @In
     private Config config;
 
+    @In
+    private ModuleManager moduleManager;
+
+    /**
+     * Determine if command is matching one of criteria
+     *
+     * @param searchLowercase searched string
+     * @param command         ConsoleCommand to check if matches searched string
+     * @return boolean containing true if command matches searched string else false
+     */
+    private static boolean matchesSearch(String searchLowercase, ConsoleCommand command) {
+        return command.getName().toLowerCase().contains(searchLowercase)
+                || command.getDescription().toLowerCase().contains(searchLowercase)
+                || command.getHelpText().toLowerCase().contains(searchLowercase)
+                || command.getUsage().toLowerCase().contains(searchLowercase)
+                || command.getRequiredPermission().toLowerCase().contains(searchLowercase);
+    }
+
+    /**
+     * Determine if prefab is matching one of criteria
+     *
+     * @param searchLowercase searched String
+     * @param prefab          Prefab to check if matches searched string
+     * @return boolean containing true if prefab matches searched string else false
+     */
+    private static boolean matchesSearch(String searchLowercase, Prefab prefab) {
+        return prefab.getName().toLowerCase().contains(searchLowercase)
+                || prefab.getUrn().toString().toLowerCase().contains(searchLowercase);
+    }
+
+    /**
+     * Determine if block family matches one of criteria
+     *
+     * @param searchLowercase searched string
+     * @param def             BlockFamilyDefinition to be checked
+     * @return boolean containing true if blockFamilyDefinition matches searched string else false
+     */
+    private static boolean matchesSearch(String searchLowercase, BlockFamilyDefinition def) {
+        return def.getUrn().toString().toLowerCase().contains(searchLowercase);
+    }
+
```

Comment:
bleh

Reply:
What happened?

Reply:
Irrelevant, unnecessary code move

Reply:
> bleh

I hereby award @nihal111 with the "Most constructive and informative code review comment ever" award 😁